src/newgrf_config.cpp

```diff
@@ -57,19 +54,17 @@ GRFConfig::GRFConfig(const GRFConfig &config) :
 	flags(config.flags & ~(1 << GCF_COPY)),
 	status(config.status),
 	grf_bugs(config.grf_bugs),
-	param(config.param),
-	num_params(config.num_params),
 	num_valid_params(config.num_valid_params),
 	palette(config.palette),
+	has_param_defaults(config.has_param_defaults),
 	param_info(config.param_info),
-	has_param_defaults(config.has_param_defaults)
+	param(config.param)
 {
 }
 
-void GRFConfig::SetParams(const std::vector<uint32_t> &pars)
+void GRFConfig::SetParams(std::span<const uint32_t> pars)
```

Comment:
span const, or const span? or const span const? :)

Reply:
It's now a span of const elements.